### PR TITLE
DIRECTOR: Add detection for Jan Lindblad presenterar den sjungande Fågelboken

### PR DESCRIPTION
**Pull request by eriktorbjorn**
This is my attempt at adding detection for Jan Lindblad presenterar den sjungande Fågelboken:

image

(The original price tag says 595.00 SEK, or about $56. I paid quite a bit less for it at my local thrift store!)

It starts, which is encouraging:

image

But while you can click on the buttons, and they depress, they never pop back up and nothing else happens. There is also a Macintosh version on the CD, but extracting it with the Dumper Companion still gives you the Windows EXE in addition to the Mac one, so I don't know if there's any need for a separate detection entry for it? Though a bunch of the files get extracted as MacBinary, so maybe it is needed after all?

### CODE REVIEW

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -419,6 +419,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "shrmess",			"Schoolhouse Rock!: 1st-4th Grade Math Essentials" },
 	{ "simpsonsplayer",		"The Simpsons Cartoon Player" },
 	{ "simpsonsstudio",		"The Simpsons Cartoon Studio" },
+	{ "singingbirdbook",		"Jan Lindblad presenterar den sjungande Fågelboken" },
```
Comment (tag2015) [inline engines/director/detection_tables.h:422]: If this is swedish-only it should be listed among the swedish titles (line 1395)

File: engines/director/detection_tables.h [hunk 2]
```diff
@@ -2260,6 +2261,13 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINDEMO2t("wrath", "Demo", "WRATH.EXE",	 "5d0ee796571b99d402a06438ae2f3d56", 696815,
 							   "SHARED.DIR", "e709f7b88f6241e4f45632beb0533ac1", 3570542, 404),
 
+	// Jan Lindblad presenterar den sjungande Fågelboken
+	// There is also a Macintosh version which also appears to be using
+	// Macromedia 4.0.4. But dumping that version still gives you the
+	// lindblad.exe file, and I assume all the data files are functionally
+	// the same, even if they have resource and data forks.
+	WINGAME1t_l("singingbirdbook", "", "lindblad.exe", "713376834e4e94d87ad931a66c3575fc", 698995, Common::SE_SWE, 404),
+
```
Comment (tag2015) [inline engines/director/detection_tables.h:2270]: If there's a mac director executable we usually add it anyway.
- The entry should be moved after line 3337, as this is not an officially supported game
- For new entries we indicate the tail in the checksum:
`WINGAME1_l("singingbirdbook", "", "lindblad.exe", "t:713376834e4e94d87ad931a66c3575fc", 698995, Common::SE_SWE, 404),`

Comment (eriktorbjorn) [reply engines/director/detection_tables.h]: Oops, I hope it's better now!

**Comment by sev-** [comment]
Thank you! The buttons and script will require further investigation

**Pull request merged**